### ISSUE TITLE
Error when writing id3 tags to big mp3 files

### ISSUE THREAD

**Issue by garciadelcastillo** [Zazama/node-id3#7]
Hi there!

I wrote a Node.js application that batch downloads a series of podcasts, and uses `node-id3` to overwrite the mp3's tags with dates, titles, etc. The application can be found here: [https://github.com/garciadelcastillo/Documentos-RNE](https://github.com/garciadelcastillo/Documentos-RNE)

The module works perfect when used on small files. However, when applying this on +50Mb files, I get this error:
```
buffer.js:74
      throw new Error(
      ^

Error: If encoding is specified then the first argument must be a string
    at new Buffer (buffer.js:74:13)
    at NodeID3.createTextFrame (D:\Dropbox\Jose Luis\code\Documentos-RNE\node_modules\node-id3\index.js:229:25)
    at NodeID3.write (D:\Dropbox\Jose Luis\code\Documentos-RNE\node_modules\node-id3\index.js:59:30)
    at WriteStream.<anonymous> (D:\Dropbox\Jose Luis\code\Documentos-RNE\app.js:205:28)
    at emitNone (events.js:91:20)
    at WriteStream.emit (events.js:185:7)
    at finishMaybe (_stream_writable.js:515:14)
    at afterWrite (_stream_writable.js:389:3)
    at onwrite (_stream_writable.js:379:7)
    at WritableState.onwrite (_stream_writable.js:90:5)
```

Some googling led me to trying with older versions of Node. The error showed up with v6.9.2. The module works like a charm when downgrading to Node v.5.12.0! 

I'd be happy to help or answer more questions when possible. 

Thanks for a super helpful module! :)

**Comment by Zazama** [comment]
Thanks for reporting.

I've run some tests and this error appears because the year is no string. I'll push a fix before the end of the year as version 0.0.8.

P.S. For me, it also happens on smaller files.

**Comment by garciadelcastillo** [comment]
Perfect, working great now. Great job with the module ;)
